src/main/java/frc/lib/motors/MotorOutput.java

```diff
@@ -0,0 +1,72 @@
+package frc.lib.motors;
+
+import static edu.wpi.first.units.Units.Amps;
+import static edu.wpi.first.units.Units.Rotations;
+import static edu.wpi.first.units.Units.RotationsPerSecond;
+import static edu.wpi.first.units.Units.RotationsPerSecondPerSecond;
+import static edu.wpi.first.units.Units.Volts;
+
+import java.util.function.Supplier;
+
+import com.ctre.phoenix6.controls.ControlRequest;
+
+import edu.wpi.first.units.measure.Angle;
+import edu.wpi.first.units.measure.AngularAcceleration;
+import edu.wpi.first.units.measure.AngularVelocity;
+import edu.wpi.first.units.measure.Current;
+import edu.wpi.first.units.measure.Voltage;
+import frc.lib.configs.MechanismConfig;
+
+/** Simple interface that abstracts motor hardware (allowing us to sent a control request to a motor and get values back, no matter if it's one motor, two motors, or a simulated motor) */
+public interface MotorOutput {
+  
+  //TODO: not sure if it's better to handle values like we did before, with a values class in the subsystem that is changed by the controller, or maybe try something new like this unchangin object of suppliers that gets returned directly from the controller
+  /** Class that provides references to value suppliers from the real or simulated output motor(s) */
+  public static class MotorValues {
```

Comment:
For consistency with future **ThingValues** types, maybe move this to different file / package.